### DISCUSSION TITLE
05/02/22 meeting Notes

### DISCUSSION THREAD

**Discussion by mrmetaverse** [AngellXR/metaversedev#14] [General]
## metaverse.dev 
:tv: [Recording](https://drive.google.com/file/d/11zWJoLLe4xo5ltdhpV_mljZVsv4kq63I/view)
- team met today at 5:05ET 21:05UTC 
- Shirley can't make it, sending AI Bot instead 
- Mike Leffer can't make it today either 
- I made a tweet to announce the effort.

### Meeting notes: 
<b>Attendees </b>

Cruz
Daniel
James
Andy
Jesse A
Michael
Joseph
Julius V
Jesse J

- Backstory on Metaversity
- Github background and run through
	- Went over discussions and general
	- Issues etc.
	- Projects and kanban board
- Project lead opportunities proposed
	- Jesse A will still be leading this effort for now

	
- talked about why this work matters 
- intros: who you are, what you do, why this matters 
- self serving for platforms because they can promote their work, and farm new candidates 
- self serving for creators to establish themselves as authority

**Comment by mrmetaverse** [comment]
I think we should find a new time to meet (demo)

**Comment by mrmetaverse** [comment]
https://github.com/AngellXR/metaversedev/issues/6 

James will look into different licenses

**Comment by mrmetaverse** [comment]
we reviewed the [readme](https://github.com/AngellXR/metaversedev), the [github strategy](https://hackmd.io/@mrmetaverse/omi-github-process), and [milestones](https://github.com/AngellXR/metaversedev/milestones)

**Comment by mrmetaverse** [comment]
@jsejms you said you took notes as well right? Can you paste here? <3 :eyes:

**Reply by jsejms** [reply]
Its pasted up top!
